39807.9 <c-34#234> Created logger
39807.9 <c-34#234> appl_initialize was called
39807.9 <c-34#234:9> Logger was updated
39807.9 <c-34#234:9> Started intializing cube lib
39807.9 <c-34#234:9> Finished intializing cube lib with worker, starting to work
39808 <c-34#234:9> requestCubes => [16] Buffer size: 0
39808 <c-34#234:9> Sent requestCubes signal to root
39808 <c-34#234:9> Received 1 cubes from root
39808 <c-34#234:9> The main loop continues
39808 <c-34#234:9> Started solving a cube
39866 <c-34#234:9> Cube failed
39866 <c-34#234:9> Added failed cube
39866.1 <c-34#234:9> returnFailedCubes => [16] Buffer size: 3
39866.1 <c-34#234:9> Sent 1 failed cubes to root
39866.1 <c-34#234:9> Received receivedFailedCubes signal from root
39866.1 <c-34#234:9> requestCubes => [16] Buffer size: 0
39866.1 <c-34#234:9> Sent requestCubes signal to root
39866.1 <c-34#234:9> Received 1 cubes from root
39866.1 <c-34#234:9> The main loop continues
39866.1 <c-34#234:9> Started solving a cube
40789.3 <c-34#234:9> appl_interrupt was called
40789.3 <c-34#234:9> Interrupting worker
40789.3 <c-34#234:9> appl_interrupt was called
40789.3 <c-34#234:9> appl_withdraw was called
40789.3 <c-34#234:9> Started withdraw thread
40789.3 <c-34#234:9> STOP (0.00s since last cb)
40789.3 <c-34#234:9> Solving interrupted
40789.3 <c-34#234:9> The main loop continues
40789.3 <c-34#234:9> Leaving the main loop due to an interruption
40789.3 <c-34#234:9> Finished withdrawing lib
40789.3 <c-34#234:9> Enter destructor of CubeWorker
40789.3 <c-34#234:9> Time waiting for messages: 0.000
40789.3 <c-34#234:9> Enter destructor of CubeWorkerInterface
40789.3 <c-34#234:9> Enter destructor of CubeRoot
40789.3 <c-34#234:9> Finished withdraw thread
40852.1 <c-34#234:9> Enter destructor
40852.1 <c-34#234:9> Time working: 981.340
40852.1 <c-34#234:9> Time suspended: 0.000
40852.1 <c-34#234:9> Joined cleanup thread
40852.1 <c-34#234:9> Exit destructor
40852.1 <c-34#234:9> Destructing logger
